README.md

```diff
@@ -43,22 +45,34 @@ Setup Instructions:
 9. $ npm run seed-sample-assets
 10. Navigate your browser to localhost:3000/asset
 
+Optionally run data feeds to update the assets (watch maps and graphs update in the browser):
+11. $ npm run update-sample-assets
+
+Shutdown Instructions:
+1. Navigate to sawtooth-supply-chain directory 
+2. $ docker-compose down
+
+Subsequent Runs:
+1. $ docker-compose up
+2. Open a new terminal
+3. Navigate to sawtooth-supply-chain/server
+4. $ npm run make-asset
+5. $ npm run seed-sample-assets
+6. Navigate your browser to localhost:3000/asset
+7. $ npm run update-sample-assets
```

Comment:
I would suggest having this be step 6

Reply:
If you wanted to view the assets updating live, you would want to navigate there first